transform/snowflake-dbt/models/blapi/source.yml

```diff
@@ -21,7 +21,6 @@ sources:
       - name: credit_cards_version
       - name: customers
       - name: customers_version
-      - name: features
       - name: features_version
```

Comment:
do we still need this one?

Reply:
Thanks @nickmisasi !

This list contains all the table in the respective schema, including unused tables. Going over tables used right now, will remove the rest in the end.